### DISCUSSION TITLE
Where does RATH stores its internal data?

### DISCUSSION THREAD

**Discussion by diogoideguchi** [Kanaries/Rath#361] [Q&A]
Hi, everyone.

I'm using the docker-compose provided in the repo to bring up RATH locally. Out of curiosity, while using docker, where does RATH stores its data? For example if I create some dashboards or analysis, I want it to be persisted between machine reboots. Any thoughts about putting a docker volume in the compose file to support it or an external database like MySQL/Postgres/Mongo/Redis?

**Comment by ObservedObserver** [comment]
Sorry for responding late. Not familiar with the discussion module in github.

RATH stores its internal data in the indexedDB in the users' browser. The server side cannot get the data for now.
